src/main/java/de/bonndan/nivio/output/map/svg/SVGItemLabel.java

```diff
@@ -27,11 +27,10 @@ public DomContent render() {
 
         ContainerTag labelText = new SVGLabelText(name, "0", 2 * DEFAULT_ICON_SIZE + 30 + "", "itemLabel").render();
 
-        ContainerTag g = SvgTagCreator.g(null, labelText).attr("class", "label");
-        g.attr("id", id);
-        g.attr("data-identifier", identifier);
-        g.attr("text-anchor", "middle");
-        return g;
+        return SvgTagCreator.g(null, labelText)
+                .attr("class", "label")
+                .attr("id", id)
+                .attr("text-anchor", "middle");
```

Comment:
What does removing the "data-identifier" attribute do?

Reply:
"data-identifier" is used in javascript to obtain the item identifier. This has been moved to a parent element (see below).